enable/__init__.py

```diff
@@ -20,6 +20,13 @@
     __requires__.append('pillow')
 
 __extras_require__ = {
-    'demo': ['chaco', 'mayavi', 'scipy', 'traitsui[demo]'],
+    'demo': [
+        'chaco',
+        'mayavi',
+        'scipy',
+        'traitsui[demo]',
+        'kiwisolver',
+        'pyglet'
+    ],
```

Comment:
> Do you think we can take this opportunity to replace "traitsui[demo]" to just "traitsui"?
`traitsui` alone already exists under `__requires__`, so maybe this can be removed entirely?  If it stays it should definitely be replaced by `traitsui[examples]`.

Reply:
> traitsui alone already exists under __requires__, so maybe this can be removed entirely?

Yes you are right. Please just remove `traitsui[demo]` entirely.

Reply:
> rename "demo" to "examples" too

I just realized that this might be confusing as the word `demo` appears in two places. Sorry about that. I meant renaming the "demo" in the *key* of the dictionary, not the "demo" in the `traitsui[*]` value.